python/searchSystemPackagePPSM.py

```diff
@@ -0,0 +1,31 @@
+# list the ports, protocols, and services in a system package
+# ex: python3 searchSystemPackagePPSM.py http://192.168.13.111:8080 companyinfra openrmfprosvc hvs.xxxxxxxxxxxxxx
```

Comment:
Need to update the comment here to pass in that port number like `8443` or something